src/core/src/op/softsign.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/softsign.hpp"
+
+#include <ngraph/validation_util.hpp>
+
+#include "itt.hpp"
+#include "ngraph/attribute_visitor.hpp"
+#include "ngraph/op/util/op_types.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+#include "ngraph/runtime/reference/softsign.hpp"
+
+namespace {
+template <ov::element::Type_t ET>
+inline bool evaluate(const ov::HostTensorPtr& arg, const ov::HostTensorPtr& out, const size_t count) {
+    using T = typename ov::element_type_traits<ET>::value_type;
+    ngraph::runtime::reference::softsign<T>(arg->get_data_ptr<ET>(), out->get_data_ptr<ET>(), count);
+    return true;
+}
+
+bool evaluate_softsign(const ov::HostTensorPtr& arg, const ov::HostTensorPtr& out) {
+    bool rc = true;
+    size_t count = shape_size(arg->get_shape());
+
+    switch (arg->get_element_type()) {
+        NGRAPH_TYPE_CASE(evaluate_softsign, bf16, arg, out, count);
+        NGRAPH_TYPE_CASE(evaluate_softsign, f16, arg, out, count);
+        NGRAPH_TYPE_CASE(evaluate_softsign, f32, arg, out, count);
+        NGRAPH_TYPE_CASE(evaluate_softsign, f64, arg, out, count);
+    default:
+        rc = false;
+        break;
+    }
+    return rc;
+}
+
+}  // namespace
+
+// *** SOFTSIGN OP SET V9 **
+BWDCMP_RTTI_DEFINITION(ov::op::v9::SoftSign);
+
+ov::op::v9::SoftSign::SoftSign(const Output<Node>& arg) : Op({arg}) {
```

Comment:
please inherit from base class for all activation operations like `UnaryElementwiseArithmetic`

Reply:
In this case you need to have BWD macros. Please don't remove it if you change base class